Create Basic Take Action Page

vkoves opened the pull request:
# Description

Added a new take action page, with an alderperson lookup that lets users input their addresses an use the Google Maps Places API to find the related GPS coordinates, then use a library to find the matching ward that it is in. **Note that the new banner appears on all pages, _including buildings_** to help guide first time visitors coming directly in from sites like Google.

| Take Action Page | Banner (all pages but Take Action) |
| --- | --- |
| <img width="3532" height="2636" alt="image" src="https://github.com/user-attachments/assets/bb44803d-76d2-4863-9a2c-d16e0d8e2259" /> | <img width="3577" height="1981" alt="image" src="https://github.com/user-attachments/assets/fcee9849-248a-44bf-83e0-0bcb748ee9dc" /> |

Additionally, I improved the Ward and Wards pages by adding the alder lookup to the Ward page (making it easy to find your ward if you don't know it) and adding alders to each ward page.

| Wards With Search | Ward Sample |
| --- | --- |
| <img width="3585" height="3071" alt="image" src="https://github.com/user-attachments/assets/30db0df0-3e79-441d-b2d7-e115b0d7c35b" /> | <img width="3585" height="3071" alt="image" src="https://github.com/user-attachments/assets/716a1f91-6282-4d97-a8c8-7e08f9baf514" /> |

### TODOS

- [x] Add take action banner to homepage
- [x] Make emails pre-populate with body and subject, or reuse email modal
- [x] Add social image / optimize metadata

Fixes #239 

# Testing Instructions

Manual QA, with automated tests for the new scraping tool. For simplicity I made it so you can quickly jump to a ward by going to `/act?ward=NUM` (e.g. `/act?ward=3`), but it's important to validate a few different addresses to make sure the ward lookup is right. The [councilmatic lookup](https://chicago.councilmatic.org/council-members/) is a good cross-reference.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If I added a large new feature, I added it to the release notes (ReleaseNotes.vue)

## Data Update (if applicable):

- [ ] I have followed the [Data Update Checklist](DATA_UPDATE_CHECKLIST.md) for updating to new year's data

<!-- PR template modified from: https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/ -->

vkoves commented on src/components/WardLookup.vue:
```diff
@@ -0,0 +1,611 @@
+
+      // If we get here, the point wasn't in any ward
+      this.error =
+        'This address does not appear to be in Chicago. Please enter a Chicago address.';
```
**TODO:** Move errors into constants

vkoves commented:
I noticed the city website has different emails than the city clerk, at custom domains, and I think these are likely to be more accurate ways of reaching folks: https://www.chicago.gov/city/en/about/wards.html

Some of these (like Beale) are just the city email, but several (40th, 47th, and 1st as examples) are not.

hunter-yeago commented on src/google-maps-loader.vue:
```diff
@@ -0,0 +1,92 @@
+ */
+export async function loadGoogleMaps(apiKey: string): Promise<void> {
+  // Check if already loaded
+  if ((window as any).google?.maps) {
```
🪟🪟🪟🪟🪟🪟  🪟   🪟  🪟🪟🪟🪟  🪟🪟🪟🪟🪟   🪟🪟🪟🪟🪟   ❓
  🪟      🪟 🪟  🪟   🪟      🪟         🪟      ❓❓
  🪟       🪟     🪟🪟🪟🪟   🪟🪟🪟     🪟🪟🪟🪟 
  🪟       🪟     🪟        🪟         🪟         ❓
  🪟       🪟     🪟        🪟🪟🪟🪟🪟   🪟🪟🪟🪟🪟   ❓

hunter-yeago commented on src/templates/Ward.vue:
<img width="1352" height="893" alt="Screenshot 2025-11-04 at 8 35 51 PM" src="https://github.com/user-attachments/assets/e719f48a-6c4d-4270-aff7-0be887a1bcfb" />
<img width="613" height="869" alt="Screenshot 2025-11-04 at 8 35 59 PM" src="https://github.com/user-attachments/assets/5d1a9deb-aa34-4754-a786-d9641e6b18c6" />

~~Buttons are a touch incongruent~~

hunter-yeago commented on src/utils/alder-data.vue:
```diff
@@ -0,0 +1,163 @@
+export interface WardFeature {
+  type: 'Feature';
+  properties: WardProperties;
+  geometry: any;
```
A^2 + B^2 = C^2

hunter-yeago commented on src/pages/Act.vue:
<img width="1496" height="885" alt="Screenshot 2025-11-04 at 8 51 08 PM" src="https://github.com/user-attachments/assets/9eda1eb2-362d-47d9-a9e5-326b7a7b086a" />

probably adjust the padding here a smidge

hunter-yeago reviewed (commented):
1. I highlighted the "any" messages as requested.
2. Act Page buttons could use a little TLC for responsivity and design (consistent padding / font size / etc)
3. Act Page - awkward spacing between two parts of the copy

All of this stuff can wait for a second PR if we want to get this up fast. All of the functionality on the frontend looks great!

hunter-yeago commented on src/constants/alder-email-content.constant.vue:
```diff
@@ -0,0 +1,68 @@
+      "the benchmarking ordinance. Since an inspector's salary would be only a small " +
+      'fraction of the nearly $6 million in potential fines they could collect annually, ' +
+      'we expect it to be a revenue-generating expenditure in coming years. Enforcing the ' +
+      'ordinance would also increase accountability and support voluntary climate action ' +
```
If we cut out the previous sentence we can just put the "adds revenue" as part of this list.

In that case we could also drop the word "also".

Can also replace "would" with "will".

hunter-yeago commented on src/constants/alder-email-content.constant.vue:
```diff
@@ -0,0 +1,68 @@
+      'averages to nearly $6 million in uncollected fines every year, and compliance is ' +
+      'decreasing.',
+
+    'Starting January 1, 2025, authority over the ordinance shifted to the Department ' +
```
Do we need to mention the authority shifting? Not sure what it adds.

hunter-yeago reviewed (commented):
email copy updates

vkoves commented:
@hunter-yeago - I think I've responded to all your feedback! The alder UI is improved and more responsive, I updated the email text, and I also added the banner to more pages (see Slack for reasoning).

| Ward Improvement | Building Page |
| --- | --- |
| <img width="1006" height="790" alt="Screenshot from 2025-11-04 23-03-44" src="https://github.com/user-attachments/assets/7eb23e96-aded-4186-9ba8-345eea9c3560" /> | <img width="1452" height="1000" alt="Screenshot from 2025-11-04 23-37-55" src="https://github.com/user-attachments/assets/6b175137-7f99-469f-bfa3-d50452de6fb5" /> |

hunter-yeago reviewed (approved):
![doge](https://github.com/user-attachments/assets/a321b58c-9ec1-4f57-bf3d-f891a68a3c66)
